rpcs3/emucore.vcxproj

```diff
@@ -51,6 +51,12 @@
       <Command>cmd.exe /c "$(SolutionDir)\Utilities\git-version-gen.cmd"</Command>
       <Message>Updating git-version.h</Message>
     </PreBuildEvent>
+    <Lib>
+      <AdditionalDependencies Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">ntdll.lib</AdditionalDependencies>
+    </Lib>
+    <Lib>
+      <AdditionalDependencies Condition="'$(Configuration)|$(Platform)'=='Release|x64'">ntdll.lib</AdditionalDependencies>

```

Comment:
Is this change related to
https://github.com/llvm/llvm-project/commit/cc7a18c18011d1e0c70187ceb60e3e69bf7cd0ee
When I mentioned LLVMSupport.lib; ntdll.lib and unresolved linker error
https://github.com/RPCS3/rpcs3/pull/16607#issuecomment-2612578912
I am just trying to learn more about why this change is needed and how to identify it

Reply:
no idea, but apparently it's needed now for some reason